lib/content-services/src/lib/dialogs/folder.dialog.scss

```diff
@@ -32,3 +32,15 @@
 .adf-dialog-action-button:disabled > span {
     color: var(--theme-text-color);
 }
+
+@media screen and (max-width: 380px) {
+    .adf-font-size-input {
```

Comment:
same with dialog, you can just set `mat-form-field` style here and avoid having extra css class

Reply:
noted and resolevd